code/_onclick/hud/alert.dm

```diff
@@ -245,6 +246,51 @@ or shoot a gun to move around via Newton's 3rd Law of Motion."
 	icon_state = "blobbernaut_nofactory"
 	alerttooltipstyle = "blob"
 
+// BLOODCULT
+
+/obj/screen/alert/bloodsense
+    name = "Blood Sense"
+    desc = "Allows you to sense blood that is manipulated by dark magicks."
+    icon_state = "blood"
+    var/image/finder
+
+/obj/screen/alert/bloodsense/New()
+	..()
+	finder = new('icons/mob/screen_alert.dmi', "finder_center")
+	add_overlay(finder)
+	START_PROCESSING(SSprocessing, src)
+	process()
+
+/obj/screen/alert/bloodsense/Destroy()
+    qdel(finder)
+    finder = null
+    STOP_PROCESSING(SSprocessing, src)
+    return ..()
+
+/obj/screen/alert/bloodsense/process()
+	if(!blood_target)
+		return
+	var/turf/Q = get_turf(blood_target)
+	var/turf/A = get_turf(mob_viewer)
+	if(Q.z != A.z) //The target is on a different Z level, we cannot sense that far.
```

Comment:
Wouldn't blood sense be removed if the mob_viewer was removed? I'll try to think of a solution.